src/framework/theme/components/sidebar/sidebar.component.ts

```diff
@@ -121,8 +121,8 @@ export class NbSidebarComponent implements OnInit, OnDestroy {
   private alive = true;
 
   @HostBinding('class.fixed') fixedValue: boolean = false;
-  @HostBinding('class.right') rightValue: boolean = false;
-  @HostBinding('class.left') leftValue: boolean = true;
+  @HostBinding('class.start') startValue: boolean = true;
```

Comment:
What should I do If I want to have the sidebar on the left side all the time? I completely agree this is a rare situation but how will I solve it?